### PR TITLE
inogre the vscode caches

### PR DESCRIPTION
**Pull request by xauthulei**
Signed-off-by: Leilei Hu <leileihu@leileis-mbp.cn.ibm.com>

@qiujian16 Could you help review this, Thanks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,4 +24,5 @@ test/out
 
 .idea
 .DS_Store
+.vscode
 launch.json
```
Comment (qiujian16) [inline .gitignore:28]: eof

Comment (xauthulei) [reply .gitignore]: @qiujian16 Updated, Thanks

**Comment by xauthulei** [comment]
Thanks @qiujian16

**Pull request merged**